lib/get-specs-from-issue.js

```diff
@@ -1,39 +1,117 @@
+const _ = require('lodash');
+
+var specsBlockTopLevelHeader = null;
```

Comment:
This global module variable shouldn't be global. Since you're joining the `findStart...` and `findEnd...` functions, keep this inside the `getBlock...` function. global variables should be used only for things that are global to the **module**. Not the action performed.

Reply:
Also, if you decided to share this in multiple functions, pass it from one another instead